ludwig/experiment.py

```diff
@@ -64,6 +64,7 @@ def experiment_cli(
         skip_save_test_statistics=False,
         skip_collect_predictions=False,
         skip_collect_overall_stats=False,
+        skip_save_predictions=True,
```

Comment:
All the other `_skip_save_` are `False`, but this one is `True`.  Is there a reason for that?

Reply:
Not that I can recall...probably an artifact of early testing...I changed to make default consistent with the other `skip_` parameters.